projects/ADuCM3029_demo_cn0410/Sensors/adi_cn0410.cpp

```diff
@@ -0,0 +1,216 @@
+/***************************************************************************//**
+ *   @file   adi_cn0410.cpp
+ *   @brief  Implementation of CN0410 dac interface.
+ *   @author Mircea Caprioru (mircea.caprioru@analog.com)
+********************************************************************************
+ * Copyright 2018(c) Analog Devices, Inc.
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *  - Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ *  - Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ *  - Neither the name of Analog Devices, Inc. nor the names of its
+ *    contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *  - The use of this software may or may not infringe the patent rights
+ *    of one or more patent holders.  This license does not release you
+ *    from the requirement that you obtain separate licenses from these
+ *    patent holders to use this software.
+ *  - Use of the software either in source or binary form, must be run
+ *    on or directly connected to an Analog Devices Inc. component.
+ *
+ * THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+
+#include "adi_cn0410.h"
+#include "adi_cn0410_cfg.h"
+
+namespace adi_sensor_swpack {
+
+uint8_t gpioMemory[ADI_GPIO_MEMORY_SIZE];
+
+	CN0410::CN0410()
+	{
+		//Constructor for CN0410
+	}
+
+	int8_t CN0410::Init()
+	{
+		//Initialization for SPI and GPIO
+		ADI_SPI_RESULT eSpiResult;
+		int8_t i8Result;
+
+		/* Initialize SPI driver */
+		if ((eSpiResult = adi_spi_Open(CN0410_SPI_DEV_CFG , m_spi_memory, ADI_SPI_MEMORY_SIZE, &m_spi_handle)) != ADI_SPI_SUCCESS)
+		{
+			i8Result = -1;
+		}
```

Comment:
Open bracket should be on the same line with the if and separated by space. But in this case maybe brackets are unnecessary because the if guards only one statement.